mollie-payments-for-woocommerce/includes/mollie/wc/gateway/abstract.php

```diff
@@ -844,11 +870,233 @@ protected function handlePayedOrderWebhook( $order, $payment ) {
+
+			// Get possibly already processed refunds
+			if ( $order->meta_exists( '_mollie_processed_refund_ids' ) ) {
+				$processed_refund_ids = $order->get_meta( '_mollie_processed_refund_ids', true );
+			} else {
+				$processed_refund_ids = array ();
+			}
+
+			Mollie_WC_Plugin::debug( __METHOD__ . ' Already processed refunds for ' . $log_id . ': ' . json_encode( $processed_refund_ids ) );
+
+			// Order the refund arrays by value (refund ID)
+			asort( $refund_ids );
+			asort( $processed_refund_ids );
+
+			// Check if there are new refunds that need processing
+			if ( $refund_ids != $processed_refund_ids ) {
+				// There are new refunds.
+				$refunds_to_process = array_diff( $refund_ids, $processed_refund_ids );
+				Mollie_WC_Plugin::debug( __METHOD__ . ' Refunds that need to be processed for ' . $log_id . ': ' . json_encode( $refunds_to_process ) );
+
+			} else {
+				// No new refunds, stop processing.
+				Mollie_WC_Plugin::debug( __METHOD__ . ' No new refunds, stop processing for ' . $log_id );
+
+				return;
+			}
+
+			$data_helper = Mollie_WC_Plugin::getDataHelper();
+			$order       = $data_helper->getWcOrder( $order_id );
+
+			foreach ( $refunds_to_process as $refund_to_process ) {
+
+				Mollie_WC_Plugin::debug( __METHOD__ . ' New refund ' . $refund_to_process . ' processed in Mollie Dashboard for ' . $log_id . '. Order note added, but order not updated.' );
+
+				$order->add_order_note( sprintf(
+					__( 'New refund %s processed in Mollie Dashboard! Order note added, but order not updated.', 'mollie-payments-for-woocommerce' ),
+					$refund_to_process
+				) );
+
+				$processed_refund_ids[] = $refund_to_process;
+
+			}
+
+			$order->update_meta_data( '_mollie_processed_refund_ids', $processed_refund_ids );
+			Mollie_WC_Plugin::debug( __METHOD__ . ' Updated, all processed refunds for ' . $log_id . ': ' . json_encode( $processed_refund_ids ) );
+
+			$order->save();
+
+			return;
+
+		}
+		catch ( \Mollie\Api\Exceptions\ApiException $e ) {
+			Mollie_WC_Plugin::debug( __FUNCTION__ . ": Could not load refunds for $payment->id: " . $e->getMessage() . ' (' . get_class( $e ) . ')' );
+		}
+
+	}
+
+	/**
+	 * @param WC_Order                     $order
+	 * @param Mollie\Api\Resources\Payment $payment
+	 */
+	protected function processChargebacks( WC_Order $order, Mollie\Api\Resources\Payment $payment ) {
+
+		// Get order ID in the correct way depending on WooCommerce version
+		if ( version_compare( WC_VERSION, '3.0', '<' ) ) {
+			$order_id = $order->id;
+		} else {
+			$order_id = $order->get_id();
+		}
+
+		// Debug log ID (order id/payment id)
+		$log_id = 'order ' . $order_id . ' / payment ' . $payment->id;
+
+		// Add message to log
+		Mollie_WC_Plugin::debug( __METHOD__ . ' called for ' . $log_id );
+
+		// Make sure there are chargebacks to process at all
+		if ( $payment->_links->chargebacks->href == null ) {
```

Comment:
I think `chargebacks` is optional and can be undefined. Now you have a null pointer. When `chargebacks` is provided there will always be a `href` on it. 

So You have to check for `chargebacks`:

```
if ( !$payment->_links->chargebacks ) {

}
```

You probably also want to add a `return` after the debug message because you do not need to continue.

Reply:
https://github.com/mollie/WooCommerce/commit/ab5bd8873a30cc514ab291df51b8531dc0be8415